ui/widgets/nuxeo-selectivity.js

```diff
@@ -7214,16 +7214,27 @@ typedArrayTags[weakMapTag] = false;
 
       // scope styles
       this.scopeSubtree(this.$.input, true);
+
+      // attach event listener to scroller parent to hide dropdown
```

Comment:
Can you please make it a bit more explicit here that we are closing the dropdown whenever we scroll on the parent? So we can later understand that this was intended.